src/pybind/mgr/dashboard/controllers/rbd_mirroring.py

```diff
@@ -649,9 +649,10 @@ def status(self):
         # using dashboard.
         if not orch_status['available']:
             return status
-        if not CephService.get_service_list('rbd-mirror') and not CephService.get_pool_list('rbd'):
+        if not CephService.get_service_list('rbd-mirror') or not CephService.get_pool_list('rbd'):
```

Comment:
This will not hold true in the case of [one way mirroring](https://docs.ceph.com/en/quincy/rbd/rbd-mirroring/#rbd-mirroring). In one-way mirroring, the primary cluster doesn't need an rbd-mirror daemon to be present to view the page. That's why I explicitly made the change in a [previous PR.](https://github.com/ceph/ceph/pull/51255)

Reply:
Thanks for review, I thought it was a bug. Looks like intentionally done.